test/nexus_test.go

```diff
@@ -290,10 +298,22 @@ func TestNexusWorkflowRunOperation(t *testing.T) {
 
 	nc := tc.newNexusClient(t, service.Name)
 
+	link := &common.Link_WorkflowEvent{
```

Comment:
This is redundant IMHO since we're already testing adding links from a workflow.